src/components/RangeSlider/components/DualThumb/DualThumb.tsx

```diff
@@ -306,6 +306,25 @@ export default class DualThumb extends React.Component<Props, State> {
     );
   }
 
+  @autobind
+  private handleMouseDownTrack(event: React.MouseEvent) {
+    if (event.button !== 0) return;
+    const clickXPosition = this.actualXPosition(event.clientX);
+    const {value} = this.state;
+    const distanceFromLowerThumb = Math.abs(value[0] - clickXPosition);
+    const distanceFromUpperThumb = Math.abs(value[1] - clickXPosition);
+
+    const centeredXPosition = clickXPosition - THUMB_SIZE * 1.5;
+
+    if (distanceFromLowerThumb <= distanceFromUpperThumb) {
+      this.setValue([centeredXPosition, value[1]], Control.Upper);
+      registerMouseMoveHandler(this.handleMouseMoveThumbLower);
+    } else {
+      this.setValue([value[0], centeredXPosition], Control.Lower);
+      registerMouseMoveHandler(this.handleMouseMoveThumbUpper);
+    }
+  }
+
```

Comment:
I moved the method up to sit next to the other mouse event methods.

Reply:
Not too big of a deal here, but would avoid moving code around which is also changing. At a minimum it is hard for reviewers to tell what has actually changed and worst case potentially causes reviewers to miss important parts of a PR and result in shipping bad code.